src/child/components/Favourite.js

```diff
@@ -38,36 +41,29 @@ class Favourite extends Component {
         this.props.bindings.onIconClick(this.props.stockCode);
     }
 
-    addDragTarget(stockCode) {
-        const dragTarget = document.getElementById(`stock_${stockCode}`);
-        dragTarget.addEventListener('dragstart', e => {
+    onDragStart(stockCode) {
+        return e => {
             // TODO: fade out window if it's last stock
-            dragTarget.classList.add('dragging');
+            e.target.classList.add('dragging');
```

Comment:
Has this not been addressed because using `e.target` is the intention (over `this`)? If so, what is the reasoning behind this?